.gitignore

```diff
@@ -0,0 +1,4 @@
+#Excludes the test file from the repository
+test.md
+#Exclude node modules
```

Comment:
- [ ] Please make sure you add a `test.md` file locally and it is being ignored by GitHub 🙏